### ISSUE TITLE
Update all seller basic points

### ISSUE THREAD

**Issue by cesar4design** [samuelvanderwaal/metaboss#145]
Update all nfts seller basic points one comand

**Comment by Tincon1** [comment]
Is there any way to do all NFTS in the collection at once?

> I planned todo this at a later date, could look into it sooner (i.e this weekend) seeing as there is demand for it. (@samuelvanderwaal feel free to assign this issue to me if you want).
> 
> For now, a PR is about to merge (#143) that gives a command to _Update one seller fee basis points_.
> 
> I updated bulk NFTs using this command the other day, by creating a bash script that looked something like:
> 
> ```
> export kp=~/.config/solana/me.json
> metaboss update seller-fee-basis-points --keypair $kp --account FkaGhLegVmA5UZyZrJpdy3ymRdYtomsPTEySkgN9T3jW --new-seller-fee-basis-points 1000
> metaboss update seller-fee-basis-points --keypair $kp --account XCGvLVYbghZSwFKzycTmVn2k5QCFEeYGpDjJaV1rooU --new-seller-fee-basis-points 1000
> ...
> ```
> 
> Takes a little while for the script to run but got the job done.

Is this script shared anywhere? I need to update my entire collections seller fee asap

**Comment by the-mercenaries-ltd** [comment]
I haven't had time just yet to add the update all command.

But here is a video showing how to create the bulk update script for now.

https://www.loom.com/share/6f2541af474f401f8d08b66672bccf66

**Comment by Tincon1** [comment]
> I haven't had time just yet to add the update all command.
> 
> But here is a video showing how to create the bulk update script for now.
> 
> https://www.loom.com/share/6f2541af474f401f8d08b66672bccf66

Yoo! Thank you SO much for going out of your way and doing this! VERY HELPFUL we appreciate you fam!!!!

**Comment by CrowdedRoom** [comment]
Hey guys, I started out to update the seller fee on a collection of 10000 tokens, I used the method above to create a script and while functional was not ideal for the large collection size because it was slow. So, Not having much idea what I was doing, I successfully built out the functionality for seller-fee-basis-points-all which takes in a json file, similar to the update uri-all,  containing an array of objects like so.
`{
        "mint_account": "3Bz9ZCpVNQ5x............E73D386",
        "new_seller_fee_basis_points": "1000"
    },`

I compiled it locally, it is running like a champ and soooo much faster. I will try to get a PR ready to get this updated
